### ISSUE TITLE
Release for Rails 7

### ISSUE THREAD

**Issue by 0x7466** [jhund/filterrific#207]
I was wondering if this gem is compatible with Rails 7? I'm currently getting an error that it's only compatible with Rails 5 and 6.

**Comment by whistlerbrk** [comment]
I forked and modified lib/filterrific.rb to:

```
if ![7,5,6].include?(Rails::VERSION::MAJOR)
  raise "\n\nThis version of Filterrific only works with Rails 5 and 6.\nPlease see the Filterrific README for the correct version of Filterrific to use with your version of Rails!\n\n"
end
```

and everything appears to be working for me thus far but haven't played around too much as I'm upgrading an old Rails project after being a way for a bit from Ruby. I'd love to help fix any issues holding back a release I'm not seeing yet. @jhund has been maintaining this vastly important gem of the Rails ecosystem for some time solo, think people might be happy to help.

**Comment by jhund** [comment]
I just released Filterrific 5.2.3 with support for Rails 7
